srcpkgs/wayfire/template

```diff
@@ -1,15 +1,15 @@
 # Template file for 'wayfire'
 pkgname=wayfire
-version=0.7.5
-revision=2
-_utils_commit=889fdafa7b09fc6ce70313fd3f067cfc8c21a2ac
+version=0.8.1
+revision=1
+_utils_commit=08553c418f164bf5e84613d27447a32e380b75f0
 _touch_commit=8974eb0f6a65464b63dd03b842795cb441fb6403
 create_wrksrc=yes
 build_wrksrc="${pkgname}-${version}"
 build_style=meson
 configure_args="-Dprint_trace=false"
 hostmakedepends="pkg-config wayland-devel"
-makedepends="wf-config-devel wlroots0.16-devel cairo-devel pango-devel
+makedepends="wf-config-devel wlroots-devel cairo-devel pango-devel json-c++
```

Comment:
You've changed `wlroots0.16-devel` to `wlroots-devel`.

Reply:
Don't do that.

Reply:
My apologies, it should be fixed now.

Reply:
I am mistaken it looks like. A different solution is necessary.